src/modules/navigator/rtl_direct_mission_land.cpp

```diff
@@ -212,6 +212,13 @@ void RtlDirectMissionLand::setActiveMissionItems()
 		}
 
 		mission_item_to_position_setpoint(_mission_item, &pos_sp_triplet->current);
+
+		// prevent lateral guidance from loitering at a waypoint as part of a mission landing if the altitude
+		// is not achieved.
+		if (_vehicle_status_sub.get().vehicle_type == vehicle_status_s::VEHICLE_TYPE_FIXED_WING && MissionBase::isLanding()
+		    && _mission_item.nav_cmd == NAV_CMD_WAYPOINT) {
+			pos_sp_triplet->current.alt_acceptance_radius = FLT_MAX;
+		}
```

Comment:
Currently you set this in the mission and direct_misison_land. That means it wont work for RTL mission fast forward. Any reason to not put it in mission base?

Reply:
@KonradRudin The issue is that after calling the base function, the mission class is the one that actually calculates the triplet. So I cannot do it before, otherwise the triplet will be overridden.